codes/classical/properties/alternant.yml

```diff
@@ -10,8 +10,19 @@ logical: q-ary_digits
 name: 'Alternant code'
 introduced: '\cite{doi:10.1016/S0019-9958(74)80005-7}'
 
-description: 'Stub.'
+description: 'A code with parity check matrix with n columns and t roww of the form
+\( H = \begin{bmatix}
```

Comment:
Can be made into displayed eqn using   
  \begin{align}
  \begin{split}
  eqn\\
  eqn.
  \end{split}
  \end{align}
See shor_nine.yml for details.